src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -352,7 +235,7 @@ public boolean logOut() {
    *         if the user chose not to log out
    */
   public boolean logOut(boolean showPrompt) {
```

Comment:
Do we have a finer-grained alternative to logout a single account?  

I'm concerned about keeping these methods without clearly marking them as being different.  I at least find `logOut()` tossing all accounts to surprising.  I'm ok with keeping it all-accounts but it should be noted in the javadoc.